client/src/components/Modal.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import ModalContent from './ModalContent';
+import '../css-components/Modal.css';
+
+const Modal = ({ photos }) => (
+  <div className="modal-container">
+    <div className="mask" />
+    <ModalContent photos={photos} />
+  </div>
+);
+
```

Comment:
This is much cleaner than mine, but then again, my modals have a bunch more going on inside them

Reply:
They do, and don't forget that I have separate branch for all the functionality so there will be much more going on in that.